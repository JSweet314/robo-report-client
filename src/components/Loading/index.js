import React from 'react';
import loadingGif from '../../assets/images/loading.gif';
import './style.css';

const Loading = () => {
  return (
    <div className='loading-div'>
      <img src={loadingGif} alt=""/>
    </div>
  );
};

export default Loading;