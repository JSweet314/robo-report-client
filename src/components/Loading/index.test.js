import React from 'react';
import { shallow } from 'enzyme';
import Loading from './index';

describe('Loading', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Loading />);
  });

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot();  
  });
});