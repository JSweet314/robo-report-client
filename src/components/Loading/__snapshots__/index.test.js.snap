// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match a snapshot 1`] = `
<div
  className="loading-div"
>
  <img
    alt=""
    src="loading.gif"
  />
</div>
`;
